test/logging_test.rb

```diff
@@ -0,0 +1,77 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class LoggingTest < Minitest::Test
+      class TestLogger < ActiveSupport::Logger
+        def initialize
+          @file = StringIO.new
+          super(@file)
+        end
```

Comment:
Ok, I think that makes sense. I would have `@file = StringIO.new; logger = ActiveSupport::Logger.new(@file); @file.rewind; @file.read` but this hides that complexity